packages/components/table/src/table-header/event-helper.ts

```diff
@@ -172,6 +171,21 @@ function useEvent<T>(props: TableHeaderProps<T>, emit) {
 
     if (!column.sortable) return
 
+    const clickTarget = event.currentTarget
+    if (
+      hasClass(clickTarget, 'ascending') &&
+      !column.sortOrders.includes('ascending')
+    ) {
+      return
+    }
+
+    if (
+      hasClass(clickTarget, 'descending') &&
+      !column.sortOrders.includes('descending')
+    ) {
+      return
+    }
```

Comment:
This could probably be adjusted as follows, as there seem to be too many if statements at the moment.
```ts
const clickTarget = event.currentTarget
if (['ascending', 'descending'].some(str =>  hasClass(clickTarget, str) && !column.sortOrders.includes(str))) {
	return
}
```

Reply:
It works successfully, thanks for the guidance